### PR TITLE
Add wrapper around demo to give margin-left 20px

### PR DESCRIPTION
**Pull request by EWhite613**
#patch#

### CODE REVIEW

File: tests/dummy/app/pods/application/template.hbs [hunk 1]
```diff
@@ -1 +1,3 @@
-{{outlet}}
+<div class="dummy-body">
```
Comment (sglanzer) [inline tests/dummy/app/pods/application/template.hbs]: If you put this in the application, then it won't be present when you port into the documentation site - is that the intention?

Comment (EWhite613) [inline tests/dummy/app/pods/application/template.hbs]: @sglanzer The documentation site actually imports the application.hbs , replacing the outlet with the demo.hbs

Comment (sglanzer) [inline tests/dummy/app/pods/application/template.hbs]: Ah interesting, thanks for the clarification :)

**Comment by sglanzer** [comment]
Approved

**Pull request merged**